src/v2/Apps/Artwork/Server/handleArtworkImageDownload.ts

```diff
@@ -13,6 +13,6 @@ export const handleArtworkImageDownload = async ({ req, res }) => {
     if (req.user) {
       imageRequest.set("X-ACCESS-TOKEN", req.user.get("accessToken"))
     }
-    req.pipe(imageRequest, { end: false }).pipe(res)
+    res.redirect(302, imageRequest.url.replace("original", "normalized"))
```

Comment:
Rather than piping and streaming, we can just open the image in a new tab. (Not sure whats up with `pipe` right here, but this achieves the same end goal. Also worth noting that if signed in as an admin, one can right click and hit save as right on the image.)

Reply:
wait this works? doesn't require auth?

Reply:
we dont want to serve the normalized tho

Reply:
id leave this alone. normalized is a totally differnet and lossy file

Reply:
its streamed because we dont want to buffer the whole thing into memory

Reply:
> doesn't require auth?

Can't get to this spot till you've auth'd.

Reply:
or actually, that's the spot above. I don't know... we can ship this and update so folks aren't freaked out by not being able to see artwork pages occasionall

Reply:
That's fine but we definitely need to serve the original here. normalized is not the same file. Why change this?

Reply:
As mentioned in the thread `original` would only return a weird small square

Reply:
`original` definitely works — it was working fine in replication Isabelle showed me.